src/peer-connection.spec.ts

```diff
@@ -10,7 +10,124 @@ jest.mock('./connection-state-handler');
 
 const mockCreateRTCPeerConnection = mocked(createRTCPeerConnection, true);
 
+// eslint-disable-next-line jsdoc/require-jsdoc
+function constructCandidatePairStats(id: string, localCandidateId: string, state: string) {
+  return {
+    id: `${id}`,
+    timestamp: 1671091266890.878,
+    type: 'candidate-pair',
+    transportId: 'T11',
+    localCandidateId: `${localCandidateId}`,
+    remoteCandidateId: 'I+UUFv24B',
+    state: `${state}`,
+  };
+}
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+function constructLocalCandidateStats(
+  id: string,
+  protocol: string,
+  relayProtocol: string | undefined
+) {
+  return {
+    id,
+    timestamp: 1671091266890.878,
+    type: 'local-candidate',
+    transportId: 'T21',
+    isRemote: false,
+    networkType: 'vpn',
+    ip: '2001:420:c0c8:1005::456',
+    address: '2001:420:c0c8:1005::456',
+    port: 53906,
+    protocol,
+    candidateType: 'host',
+    priority: 2122197247,
+    relayProtocol,
+  };
+}
+
 describe('PeerConnection', () => {
+  describe('getCurrentConnectionType', () => {
```

Comment:
I think it would be worth adding tests for these cases:
- when no candidate pair has succeeded
- when ice state is not connected/completed

Reply:
Both cases were added already.